files/base.rb

```diff
@@ -222,5 +222,61 @@ def handler_settings
     settings[settings_key]
   end
 
+
+  ## helpers for chat handlers
+  def channel_keys
+    %w{ channel room }
+  end
+
+  def pager_channel_keys
```

Comment:
I'm not sure the whole pager_channel concept I got from nodebot is needed at all.   if you're going to config a check to page you could just as easily  add two channels to the channels config rather than one channel and one pager_channel.   

i'm starting to think i should rip out the separate pager layer.    OTH this mechanism would support easy implementation of backwards compatability  for nodebot